src/main/java/duke/command/DeleteCommand.java

```diff
@@ -10,7 +10,7 @@
 public class DeleteCommand extends Command {
     private TasksList tasksList;
     private String[] inputArray;
-    private static final String deleteMsg = "Noted. I've removed this task:\n";
+    private static final String DELETE_MSG = "Noted. I've removed this task:\n";
```

Comment:
I like that you took note of changing `deleteMsg` to `DELETE_MSG`. I noticed you have changed it for the other constants too